recipe/meta.yaml

```diff
@@ -49,7 +49,7 @@ build:
     - {{ name }} * {{ mpi_prefix }}_*
   {% endif %}
 
-  skip: True  # [win]
+  skip: True  # [win or ((tempest == 'tempest') and (osx or (linux and (mpi == 'nompi'))))]
```

Comment:
```suggestion
  skip: True  # [win or ((tempest == 'tempest') and (mpi == 'nompi'))]
```

Reply:
This would be the build we now want if we can get OSX working again.

Reply:
I'm still puzzled by that linker error. I may have to try to replicate it locally on my OSX laptop with MOAB 5.1.0 and TempestRemap 2.0.2 combo with MPI. Worst case, I should be able to provide a patch to build it cleanly. Best case, we figure out what is wrong with the build configuration and fix that.

Reply:
To test, clone my fork and run the following:
```
conda build -m .ci_support/osx_mpimpichpython3.7tempesttempest.yaml recipe
```
When errors happen, you can go into your equivalent of `~/miniconda3/conda-bld/moab_xxxxxxx/work` and run the `conda_build.sh` script to try to figure out how to get the build to work.  You can tinker with the source right there as long as you remember what you did so you can make a patch out of it later.

Reply:
Thanks, that is helpful. Will do that in an hour and let you know what I find.